### PR TITLE
Fixes #13445: Use new doc links in Rudder

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/13445

### CODE REVIEW

File: rudder-web/src/main/resources/demo-rudder-users.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
 
   Please take a look at Rudder's documentation for additional details, either from the
   application itself (upper right corner) or on
-  http://www.rudder-project.org/rudder-doc-latest/rudder-doc.html#user-management.
+  https://docs.rudder.io/reference/current/administration/users.html
```
Comment (ncharles) [inline rudder-web/src/main/resources/demo-rudder-users.xml]: this is a 404

Comment (amousset) [reply rudder-web/src/main/resources/demo-rudder-users.xml]: indeed.... this redirection looks broken now

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Pull request merged**